src/routes/lobby/LobbyView.vue

```diff
@@ -59,7 +59,9 @@
                   data-cy="edit-game-ranked-switch"
                   color="primary"
                   hide-details
+                  tabindex="0"
                   @update:model-value="setIsRanked"
+                  @keydown.enter="toggleRanked"
```

Comment:
This works, but I think it would be simpler to fire a click. Another contributor opened #1167 to address the same issue. They have an elegant one-liner for the fix, but they didn't add a test case. I think the ideal combination would be to use your test case (plus the changes to account for focusing the switch correctly) and their solution to fire a click. I've asked them to PR into your fork to combine your PRs so you can both get contribution credit for the fix.

Would that work for you? If so, stay tuned for a PR into your fork updating this part, and if it isn't done soon then we can just update your fix along with the test case to match the one-liner and use the test to confirm if that works
```suggestion
                  @keydown.enter.stop="(e) => e.target.click()"
```